flutter/shell/platform/tizen/BUILD.gn

```diff
@@ -160,7 +161,9 @@ template("embedder") {
       libs += [
         "tzsh_common",
         "tzsh_softkey",
+        "eom",
       ]
+      defines += [ "EOM_SUPPORT" ]
```

Comment:
`defines` appears for the first time in this template at line 169. You may move the whole if block (lines 158-167) to between lines 170 and 172, and also add a blank line between `libs` and `defines`.

Otherwise you can just use the `COMMON_PROFILE` macro without defining `EOM_SUPPORT` itself.

Reply:
Thanks for your comments. COMMON_PROFILE looks better.